src/ppx/grfx/vk/vk_command.cpp

```diff
@@ -290,36 +290,67 @@ void CommandBuffer::ClearRenderTarget(
     grfx::Image*                        pImage,
     const grfx::RenderTargetClearValue& clearValue)
 {
-    auto pCurrentRenderPass = GetCurrentRenderPass();
-    if (IsNull(pCurrentRenderPass)) {
+    if (!HasActiveRenderPass()) {
         return;
     }
 
-    // Make sure pImage is a render target in current render pass
-    const uint32_t renderTargetIndex = pCurrentRenderPass->GetRenderTargetImageIndex(pImage);
-    if (renderTargetIndex == UINT32_MAX) {
-        return;
+    grfx::Rect renderArea;
+    uint32_t   colorAttachment;
```

Comment:
Perhaps we can initialize colorAttachment to UINT32_MAX and if it is still UINT32_MAX after the following if/else we can log a warning. Then we wouldn't need imageFound.

Reply:
Would we want to log a warning or just return out?

Reply:
I think we can log a warning and return